.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,144 @@
+name: Bug report
+description: Create a report to help us fix something that isn't working as expected
+labels: "bug :bug:"
+body:
+- type: markdown
+  attributes: 
+    value: |
+      ## Before you begin
+      🚨 **Please do not skip instructions**🚨
+      
+      This info is essential for investigating your bug report. Issues with missing information may be closed without investigation.
+      
+      If you're uncertain about the problem, find or start a [Questions & Help](https://github.com/CommunityToolkit/WindowsCommunityToolkit/discussions/categories/questions-help) discussion, where you can get insight from the community regarding your issue. 
+      
+- type: textarea
```

Comment:
Does YAML not allow a minimum height, or at least the ability to stretch & resize the text input area?  I'm unable to resize these textareas.

Reply:
Found it: If you add `render: text` to the attributes block, for a `textarea`, you can then finally stretch these textboxes vertically:  
![image](https://user-images.githubusercontent.com/86266896/145451476-c09b1e78-83f6-4411-abe5-e68378b9af60.png)


```
- type: textarea
  id: repro-steps
  validations:
    required: true
  attributes:
     label: Steps to reproduce
     render: text
     description: Provide steps to reproduce the issue, or let us know why it can't be reproduced (e.g. more complex setup, environment, dependencies, etc...)
     placeholder: |
        A repro from the sample app expidites the process of discovering, resolving, and validating bug fixes. 
        Provide code-snippets or XAML snippets as appropriate.
```